appium/webdriver/webdriver.py

```diff
@@ -174,6 +174,27 @@ def add_command(self) -> Tuple[str, str]:
         raise NotImplementedError()
 
 
+def _get_remote_connection_and_client_config(
+    command_executor: Union[str, AppiumConnection], client_config: Optional[AppiumClientConfig]
```

Comment:
should the optional value be assigned to None by default?

Reply:
Either is fine. Just wanted to let the caller explicitly set both